tools/cldr-code/src/main/java/org/unicode/cldr/json/Ldml2JsonConverter.java

```diff
@@ -389,18 +389,39 @@ private String transformPath(final String pathStr, final String pathPrefix) {
         if (result.startsWith("//cldr/supplemental/references/reference")) {
             // no change
         } else if (strictBcp47) {
+            // Look for something like <!--@MATCH:set/validity/locale--> in DTD
             if (result.contains("localeDisplayNames/languages/language")) {
                 result = fixXpathBcp47(result, "language", "type");
             } else if (result.contains("likelySubtags/likelySubtag")) {
-                result = fixXpathBcp47(result, "likelySubtag", "from", "to");
+                if (!result.contains("\"iw\"") &&
+                    !result.contains("\"in\"") &&
+                    !result.contains("\"ji\"")) {
+                    // Special case: preserve 'iw' and 'in' likely subtags
+                    result = fixXpathBcp47(result, "likelySubtag", "from", "to");
+                } else {
+                    result = underscoreToHypen(result);
+                    logger.warning("Including aliased likelySubtags: " + result);
+                }
```

Comment:
@yumaoka @macchiati  some special code to preserve `iw`, `in`, `ji` in likelySubtags - otherwise, they end up duplicating `he`, `id`, `yi` due to aliasing.  As an alternative, these likelysubtags could be dropped.